source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -0,0 +1,248 @@
+---
+title: External Network Providers
+category: feature
+authors: mmirecki
+wiki_category: Feature
+wiki_title: Features/External Network Providers
+wiki_revision_count: 1
+wiki_last_updated: 2016-01-25
+feature_name: External Network Providers
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# External Network Providers
+
+
+## Summary
+
+The purpose of this feature is to enable support for networks not defined by oVirt, but supplied by an external provider.
```

Comment:
this feature would not *enable* this support (as it already exists via openstack network), but only to simplify it, and extend it to Neutron-like providers.